.kokoro

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+if [ ! -d .kokoro-ios-runner ]; then
+  git clone https://github.com/material-foundation/kokoro-ios-runner.git .kokoro-ios-runner
+fi
+
+pushd .kokoro-ios-runner
+git fetch > /dev/null
+TAG=$(git tag -l "v3*" | sort | tail -n1)
+git checkout $TAG > /dev/null
+popd
+
+if [ -z "$KOKORO_BUILD_NUMBER" ]; then
+  tests_dir_prefix=""
+else
+  tests_dir_prefix="github/repo/"
+fi
+
+# Fixes a bug in bazel where objc_library targets have a _ prefix.
+find ${tests_dir_prefix}tests/unit -type f -name '*.swift' -exec sed -i '' -E "s/import Motion(.+)/import _Motion\1/" {} + || true
+# CocoaPods requires that public headers of dependent pods be implemented using framework import
+# notation, while bazel requires that dependency headers be imported with quoted notation.
+find ${tests_dir_prefix}src -type f -name '*.h' -exec sed -i '' -E "s/import <MotionInterchange\/MotionInterchange\.h>/import \"MotionInterchange.h\"/" {} + || true
+stashed_dir=$(pwd)
+reset_imports() {
```

Comment:
Nit: Prefer to define functions at the top of the file (after global configs).

Reply:
Done.

Reply:
Done.